Update interfaces/html.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Note: This file was recently manually updated in commit 4c70b5a6f3.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/66323ce/whatwg/idl/html.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/498574778

foolip commented:
This would revert https://github.com/web-platform-tests/wpt/pull/15123 and shouldn't be merged. After https://github.com/whatwg/html/pull/4324 is merged it should be closed automatically if no changes remain.

autofoolip committed:
Update interfaces/html.idl

Note: This file was recently manually updated in commit 4c70b5a6f3.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/66323ce/whatwg/idl/html.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/498574778

foolip commented:
Unblocked per https://github.com/whatwg/html/pull/4324#issuecomment-477873715

foolip commented:
Taskcluster failure is because of https://github.com/web-platform-tests/wpt/issues/16175.
